timestamp-batch/src/main/java/io/spring/configuration/TimestampBatchTaskConfiguration.java

```diff
@@ -50,18 +58,46 @@
 	@Autowired
 	private TimestampBatchTaskProperties config;
 
+	/**
+	 * Override default transaction isolation level 'ISOLATION_REPEATABLE_READ' which Oracle does not
+	 * support.
+	 */
+	@Configuration
+	@ConditionalOnProperty(value = "spring.datasource.driver", havingValue = "oracle.jdbc.OracleDriver")
+	static class OracleBatchConfig {
+		@Bean
+		BatchConfigurer oracleBatchConfigurer(DataSource dataSource) {
+			return new DefaultBatchConfigurer() {
```

Comment:
No because it's @Autowired https://github.com/spring-projects/spring-batch/blob/4.3.x/spring-batch-core/src/main/java/org/springframework/batch/core/configuration/annotation/DefaultBatchConfigurer.java#L56

Reply:
Ahh good eye. This is a perfect example why random setter injection is not a recommended practice.